src/main/scala/firrtl/passes/memlib/ReplaceMemTransform.scala

```diff
@@ -96,8 +96,9 @@ Optional Arguments:
 class SimpleTransform(p: Pass, form: CircuitForm) extends Transform {
   def inputForm = form
   def outputForm = form
-  def execute(state: CircuitState): CircuitState = state.copy(circuit = p.run(state.circuit))
+  def execute(state: CircuitState): CircuitState = state.copy(circuit = p.run(state.circuit), annotations = None)
```

Comment:
copy here will still copy the rename map though right? Perhaps it should just be

```scala
def execute(state: CircuitState): CircuitState = CircuitState(p.run(state.circuit), outputForm)
```